src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -106,4 +109,31 @@ public String getByArtefactId(UUID artefactId, Boolean verification) {
             throw new NotFoundException(String.format("No artefact found with the ID: %s", artefactId));
         }
     }
+
+    /**
+     * Enforces conditional mandatory fields based on the publication type. This is to ensure that status updates are
+     * able to persist indefinitely if required.
+     */
+    public boolean validateDateFromDateTo(LocalDateTime displayFrom, LocalDateTime displayTo, ArtefactType type) {
+
+        ArrayList<ArtefactType> mandatoryDateArtefactTypes = new ArrayList<>();
+        mandatoryDateArtefactTypes.add(ArtefactType.LIST);
+        mandatoryDateArtefactTypes.add(ArtefactType.OUTCOME);
+        mandatoryDateArtefactTypes.add(ArtefactType.JUDGEMENT);
+
+        if (mandatoryDateArtefactTypes.contains(type)) {
+            if (displayFrom == null) {
+                throw new HeaderValidationException(String.format("Date from field is mandatory for publication "
+                                                                      + "type %s", type));
+            }
+            if (displayTo == null) {
+                throw new HeaderValidationException(String.format("Date to field is mandatory for publication "
+                                                                      + "type %s", type));
+            }
+        } else {
+            return !type.equals(ArtefactType.STATUS_UPDATES) || displayFrom != null;
```

Comment:
I find this line a bit confusing. Could just be me but I think I'd prefer seperating the status updates these out into an else if to make it clearer

Reply:
Should be much clearer with the new validationService